cdk/lib/support-workers.ts

```diff
@@ -369,7 +426,11 @@ export class SupportWorkers extends GuStack {
       actionsEnabled: isProd,
       snsTopicName: `alarms-handler-topic-${this.stage}`,
       alarmName: `support-workers ${this.stage} No successful recurring stripe supporter plus contributions recently.`,
-      metric: this.buildPaymentSuccessMetric("Stripe", "SupporterPlus", Duration.seconds(3600)),
+      metric: this.buildPaymentSuccessMetric(
+        "Stripe",
+        "SupporterPlus",
+        Duration.seconds(3600)
+      ),
```

Comment:
there seem to be quite a few formatting changes so far, assuming they are just automated, perhaps you can do them on another PR, and merge into this PR to make it easier to review?

Reply:
https://github.com/guardian/support-frontend/pull/6756